src/graphing/blips.js

```diff
@@ -104,9 +104,21 @@ function findBlipCoordinates(blip, minRadius, maxRadius, startAngle, allBlipCoor
 }
 
 function blipAssistiveText(blip) {
+  let blipType
+
+  if (blip.isNew()) {
+    blipType = 'new'
+  } else if (blip.hasMovedIn()) {
+    blipType = 'moved in'
+  } else if (blip.hasMovedOut()) {
+    blipType = 'moved out'
+  } else {
```

Comment:
Should the `hasNoChange` fxn be incorporated here?

Reply:
Simplified to use blip.status directly (previously the assistive text was formatted in a way that wasn't compatible.  Now, not the case :)